src/test/mir-opt/matches_u8.rs

```diff
@@ -0,0 +1,32 @@
+// EMIT_MIR_FOR_EACH_BIT_WIDTH
```

Comment:
Are our mir-opt tests running on mir-opt-level 3 by default?

Reply:
Uh, I'm not sure, but when I ran it initially with this file it failed

Reply:
I think they run with `mir-opt-level=2` but I can't find it in the source off hand.

Reply:
https://github.com/rust-lang/rust/blob/8e21bd0633b8d970646ee6eb706c9e8acfad19af/src/tools/compiletest/src/runtest.rs#L1941-L1946